### ISSUE TITLE
macOS build

### ISSUE THREAD

**Issue by mars3142** [Zal0/ZGB#64]
Where can I get a macOS build? I don't own a windows maschine.

**Comment by Zal0** [comment]
macOS is not supported yet

**Comment by mars3142** [comment]
That's why I opened the issue. Should I create a Feature Request for it?

**Comment by Zal0** [comment]
Yes, although the main issue right now is having a crossplatform tile and sprite editor, I will work on adding to support to Linux and macOS when we have that

**Comment by mars3142** [comment]
Haha. I was planning in writing one in Flutter. What are the MVP features for it?

**Comment by Zal0** [comment]
Minimum is having all the features that Tile Designer and Map Creator have, plus crossplatform support

**Comment by mars3142** [comment]
I started the project and will also deploy a web version (https://gb-designer.mars3142.dev) , so it's also operating system independent. But it's a long way to go for a MVP.

**Comment by asturur** [comment]
@mars3142 did you move forward with zgb on macos?
